app/src/main/java/com/monstarlab/core/ui/extensions/ContextExtensions.kt

```diff
@@ -0,0 +1,65 @@
+package com.monstarlab.core.ui.extensions
+
+import android.content.ClipData
+import android.content.ClipboardManager
+import android.content.Context
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.net.Uri
+import android.os.Build
+import android.provider.Settings
+import android.widget.Toast
+import androidx.core.content.ContextCompat
+import androidx.core.net.toUri
+import com.monstarlab.core.extensions.castAs
+
+fun Context.openSettings() {
+    val intent = Intent(Settings.ACTION_APPLICATION_DETAILS_SETTINGS)
+    intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
+    val uri = Uri.fromParts("package", packageName, null)
+    intent.data = uri
+    startActivity(intent)
+}
+
+fun Context.openNotificationSettings() {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+        val intent = Intent(Settings.ACTION_APP_NOTIFICATION_SETTINGS)
+            .putExtra(Settings.EXTRA_APP_PACKAGE, packageName)
+            .addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
+        startActivity(intent)
+    } else {
+        openSettings()
+    }
+}
+
+fun Context.copyToClipboard(value: String) {
+    val cm = getSystemService(Context.CLIPBOARD_SERVICE).castAs<ClipboardManager>()
+    cm.setPrimaryClip(ClipData.newPlainText("Clip", value))
```

Comment:
I would suggest adding the `"Clip"` as a parameter for the extensions so it can be customized.

Reply:
Addressed 👍